backends/p4tools/modules/testgen/targets/bmv2/test/BMV2PTFXfail.cmake

```diff
@@ -40,7 +40,8 @@ p4tools_add_xfail_reason(
 
 p4tools_add_xfail_reason(
   "testgen-p4c-bmv2-ptf"
-  "Invalid reference to object of type"
+  "Segmentation fault"
```

Comment:
This is not a good sign.
I see no reason for the changes in the test scripts to lead to a segfault, something must be wrong somewhere.

Reply:
In both cases the segmentation fault is thrown.

The underlying cause is a bit tricky. Previously, simple_switch_grpc was invoked with --log-console, which directly printed output to stdout/err. However, this occasionally overwhelmed the process pipe and the process got stuck. 

Now, we use `--log-file`, but simple switch crashes before the statement can be logged/generated. So it does not appear anymore. 

The completely solution is to either fix simple_switch's logging or to fix the Popen call so that it does not get stuck.

Reply:
If it's the same bug please file an issue and add a comment to the makefile.

Reply:
I believe it is this bug https://github.com/p4lang/behavioral-model/issues/1124

Reply:
I am not aware of a corresponding bug for p4c, we should have one. Can you please add one?

Reply:
I asked Antonin to migrate the issue.